### ISSUE TITLE
CalledProcessError. Non-Zero Exit Status.

### ISSUE THREAD

**Issue by BlakeERichey** [xflr6/graphviz#75]
I was testing a graph with the view() function and I closed the environment that called this view before the view itself. Any further attempts have resulted in this error even after turning the computer off and back on again.

```Traceback (most recent call last):

  File "<ipython-input-1-b2a737bbd5a5>", line 1, in <module>
    runfile('C:/Users/brichey/Desktop/Blake/self-assembling-process-simulator/graphviz_demo.py', wdir='C:/Users/brichey/Desktop/Blake/self-assembling-process-simulator')

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 827, in runfile
    execfile(filename, namespace)

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 110, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "C:/Users/brichey/Desktop/Blake/graphviz_demo.py", line 95, in <module>
    create_graph(None)

  File "C:/Users/brichey/Desktop/Blake//graphviz_demo.py", line 92, in create_graph
    graph.view(cleanup=True)

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\graphviz\files.py", line 229, in view
    view=True, cleanup=cleanup, quiet=quiet)

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\graphviz\files.py", line 201, in render
    quiet=quiet)

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\graphviz\backend.py", line 193, in render
    run(cmd, capture_output=True, cwd=cwd, check=True, quiet=quiet)

  File "C:\Users\brichey\AppData\Local\Continuum\anaconda3\envs\tf\lib\site-packages\graphviz\backend.py", line 159, in run
    output=out, stderr=err)

CalledProcessError: Command '['dot', '-Tpdf', '-O', 'temp.gv']' returned non-zero exit status 1.```

**Comment by BlakeERichey** [comment]
Figured out it was an HTML typo. I forgot to close a <TR> tag properly. I had <TR><TD>Data</TD>/TR>

Perhaps this may help someone in this future that gets this error message.

**Comment by xflr6** [comment]
Thanks. I think the rendering process should normally [write a more informative message to `stderr`](https://github.com/xflr6/graphviz/blob/1fe315fec8559f2955ef4001c42ae58f52a31705/graphviz/backend.py#L153-L159) (maybe it's further-up in your traceback). If you can provide a minimal isolated example, I can check if this is the case.

**Comment by xflr6** [comment]
Might want to add the `.stderr` to the `repr()` of `CalledProcessError`.

**Comment by xflr6** [comment]
Test example:
```python
>>> import graphviz
>>> d = graphviz.Digraph()
>>> d.node('<TR><TD>Data</TD>/TR>')
>>> d
Error: <stdin>: syntax error in line 2 near '/'
(...)
CalledProcessError: Command '['dot', '-Tsvg']' returned non-zero exit status 1.
```

**Comment by xflr6** [comment]
With efd944d41afc3000e09e1b30ad0e396d9622176d:

```python
CalledProcessError: Command '['dot', '-Tsvg']' returned non-zero exit status 1. [stderr: b"Error: <stdin>: syntax error in line 2 near '/'\r\n"]
```

**Comment by xflr6** [comment]
released [graphviz 0.11.1](https://pypi.org/project/graphviz/0.11.1/)
